kuka_rsi_hw_interface/launch/startup_with_rviz.launch.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2022 Áron Svastits
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from ament_index_python.packages import get_package_share_directory
+
+from launch import LaunchDescription
+from launch_ros.actions import Node
+from launch.actions.include_launch_description import IncludeLaunchDescription
+from launch.launch_description_sources.python_launch_description_source import PythonLaunchDescriptionSource  # noqa: E501
+
+
+def generate_launch_description():
+    rviz_config_file = os.path.join(
+        get_package_share_directory('kuka_agilus_support'), 'config', 'agilus_urdf.rviz')
```

Comment:
what will happen here, if user selectingthe cybertech robots? Or is it working properly just the nameing old?

Reply:
the rviz configs are the same, so that is not an issue, but maybe the two files should be moved to the driver's package to avoid this naming issue -> maybe in the next PR if that is OK :D

Reply:
okey